src/view/uniprotkb/components/VariationView.tsx

```diff
@@ -175,20 +177,22 @@ const VariationView: FC<{ primaryAccession: string }> = ({
 
   return (
     <div>
-      <h4>Variants</h4>
+      {title && <h4>{title}</h4>}
       <protvista-manager attributes="highlight displaystart displayend activefilters filters">
-        <div className="variation-view">
-          <protvista-navigation length={data.sequence.length} />
-          <protvista-sequence
-            length={data.sequence.length}
-            sequence={data.sequence}
-            height="20"
-          />
-          <protvista-filter />
-          <protvista-variation length={data.sequence.length}>
-            <protvista-variation-adapter ref={setTrackData} />
-          </protvista-variation>
-        </div>
+        {!noTable && (
```

Comment:
i'm being pedantic here but am wondering if something like `showTable` would be slightly easier to mentally compute as we're only taking the logical negation of `noTable`?

Reply:
👍